tests/phpunit/api/v3/ContributionPageTest.php

```diff
@@ -1650,6 +1654,7 @@ public static function tearDownAfterClass() {
     );
     $unitTest = new CiviUnitTestCase();
     $unitTest->quickCleanup($tablesToTruncate);
+    parent::tearDown();
```

Comment:
@eileenmcnaughton this needs to be 

```suggestion
    $unitTest->teardown();
```